business_requirement_deliverable/models/business.py

```diff
@@ -127,6 +127,13 @@ class BusinessRequirementDeliverable(models.Model):
         readonly=True,
         compute='_compute_get_currency',
     )
+    linked_project = fields.Many2one(
```

Comment:
Somewhere the information was not properly transmitted.
I said that we dont need this in the report. So no need to transfer the field to brd module